binaries/speed_benchmark_torch.cc

```diff
@@ -136,9 +140,15 @@ std::vector<c10::IValue> create_inputs() {
           "Unsupported input memory format: ", input_memory_format_list[i]);
     }
 
-    inputs.push_back(torch::ones(
+    auto input_tensor = torch::ones(
```

Comment:
Minor suggestion: consider marking this as const.  const is not implied by auto.